### DISCUSSION TITLE
Search no result

### DISCUSSION THREAD

**Discussion by makecent** [mmistakes/minimal-mistakes#3084] [Q&A] [answered]
- Git repository URL: https://github.com/makecent/makecent.github.io/tree/gh-pages/docs
- GitHub Pages hosted:  https://chongkai.site

I had made ``search: true`` in all places, but search just doesn't find anything.

**Comment by mmistakes** [comment]
Looks like you've added `search: false` to your posts. This excludes them from the search index, which is why you don't get any results.

Remove `search: false` from the YAML front matter.

**Reply by makecent** [reply]
Thanks for the reply. YAML front matters of all files are either ``search: true`` or default. Still got nothing.

**Comment by makecent** [comment, marked as answer]
I changed to use algolia. Resovled.

**Reply by mmistakes** [reply]
Good. Switching to Algolia was going to be my suggestion as it provides a better experience and more relevant results over the default Lunr.
